code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -96,7 +96,6 @@
 
 		if(ismonkey(occupant)) // Monkey
 			occupant_overlay = image(CRYOMOBS, "monkey")
-			occupant_overlay.copy_overlays(occupant)

```

Comment:
Why remove this?

Reply:
It copies the tail overlay

Reply:
You could probably `occupant_overlay.cut_overlay("<whatever the tail overlay is named>")` after the copy then so you get everything else.